students/ryba_paula/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,13 @@
+number = int(input())
+
+largest = number
```

Comment:
to jest ryzykowne, ponieważ użytkownik może podać, że wprowadzi 100 liczb i będzie podawać liczby z zakresu 1-10, to wtedy program zwróci błędny wyniki (w tym wypadku, zwróci największą liczbę np 10, zamiast 9).
Mamy zapewnione, że liczby są dodatnie, więc na początku możemy użyć coś: <=0

Reply:
@kadilak83 zmieniłam całe zadanie, jeśli coś jest nadal źle, to pisz :)